.gitignore

```diff
@@ -22,3 +22,5 @@
 
 # Test Coverage
 /coverage
+
+.rufo
```

Comment:
its a formatter right? Is that switch intended? Do every one have the same formatter and settings?

Reply:
Everyone is using Solargraph except me. I have errors using Solargraph.